src/Http/Http.Extensions/gen/StaticRouteHandlerModel/InvocationOperationExtensions.cs

```diff
@@ -27,6 +27,8 @@ public static bool IsValidOperation(this IOperation? operation, WellKnownTypes w
     {
         invocationOperation = null;
         if (operation is IInvocationOperation targetOperation &&
+            targetOperation.TargetMethod.ContainingNamespace is { Name: "Builder", ContainingNamespace: { Name: "AspNetCore", ContainingNamespace: { Name: "Microsoft", ContainingNamespace.IsGlobalNamespace: true } } } &&
```

Comment:
Seems like the assembly name check would suffice?  Or do we have our own Map methods that shouldn't be intercepted?

Reply:
https://github.com/dotnet/aspnetcore/issues/50836 is an example where a third party package used the same namespace + type name as our APIs and resulted in issues with our well-known type resolution so the additional check on assembly name is a helpful guard against those rare (but annoying) situations.

Reply:
I was asking the opposite question: why do we need the namespace check?

Reply:
> I was asking the opposite question: why do we need the namespace check?

Ah, my bad! We could get away with not having a namespace check here but I like the extra assurance it provides. 🤷🏽